index.html

```diff
@@ -12021,7 +12021,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 			<pdef>aria-level</pdef>
 			<div class="property-description">
 				<p><a>Defines</a> the hierarchical level of an <a>element</a> within a structure.</p>
-				<p>This can be applied inside trees to tree items, to headings inside a document, to nested grids, nested tablists and to other structural items that may appear inside a container or participate in an ownership hierarchy. The value for <pref>aria-level</pref> is an integer greater than or equal to 1.</p>
+				<p>This can be applied inside trees to tree items, to headings inside a document, to nested grids, nested tablists and to other structural items that may appear inside a container or participate in an ownership hierarchy. The value for <pref>aria-level</pref> is an integer greater than or equal to 1. If <pref>aria-level</pref> is being used on an element with role <pref>heading</pref> then authors MUST set the value to an integer between 1 and 6.</p>
```

Comment:
What would be the benefits of this change?

From the summary in https://github.com/w3c/aria/issues/1050#issuecomment-796481101 it seems that levels 1-9 on headings are widely supported in user agents, with only Chrome not supporting higher integer values. Similarly only JAWS appears to be failing with levels above 6 while Talkback struggles slightly with levels 7-9.

Reply:
my thoughts on this change were that as of right now all UA and AT support 1-6 and unless we suggest all UA and AT to support 1-9 then at the moment to ensure authors using `aria-level` with role heading are getting the expected experience across all UA and AT then they should use 1-6

Reply:
Thanks for clarifying. That sounds like good authoring advice (like your APG issue https://github.com/w3c/aria-practices/issues/2620). From a spec perspective, it looks like pretty good support (especially with Chrome and JAWS looking like bugs), so I personally would rather file issues with user agents and AT instead of this change.

Reply:
Tend to agree.  While I’m not itching to use heading level 8s anytime soon, I’m not sure why the spec should be so strict on that?

yes html only has up to level 6, but also, html doesn’t have buttons that natively convey a pressed state…

Reply:
@ariellalgilmore I would wait until the WG has a chance to look at the PR next week.